test/evm/operation/block_information.ex

```diff
@@ -0,0 +1,4 @@
+defmodule EVM.Operation.BlockInformationTest do
```

Comment:
should be all be `block_information_test.exs` (the `.exs` being the most important part, since it's not part of normal compilation)